doc/source/networking.rst

```diff
@@ -1,185 +1,215 @@
+There are a few steps to follow when setting up *Shared IP*:
 
-``iocage get vnet UUID | TAG``
+**Check the VNET property status**
 
-**If set to "on" disable it**
+:samp:`# iocage get vnet examplejail1`
 
-``iocage set vnet=off UUID | TAG``
+If **vnet** is on, disable it:
 
-A system wide default can be configured if required. This will take effect for newly created jails only.
-
-``iocage set vnet=off default``
+:samp:`# iocage set vnet=off examplejail1`
 
 **Configure an IP address**
 
-``iocage set ip4_addr="em0|10.1.1.10/24" UUID| TAG``
-
-If multiple addresses are desired just separate the configuration directives with a comma.
+:samp:`# iocage set ip4_addr="em0|10.1.1.10/24" examplejail1`
 
-Example:
+If multiple addresses are desired, separate the configuration directives
+with a :kbd:`,`:
 
-``iocage set ip4_addr="em0|10.1.1.10/24,em0|10.1.1.11/24" UUID| TAG``
+:samp:`# iocage set ip4_addr="em0|10.1.1.10/24,em0|10.1.1.11/24" examplejail1`
 
-**Start jail:**
+**Start the jail**
 
-``iocage start UUID | TAG``
+:samp:`iocage start examplejail1`
 
 **Verify visible IP configuration in the jail**
 
-*(jail must be running for this to work)*
+.. note:: The jail **must** be running for this example to work.
 
-``iocage exec UUID | TAG ifconfig``
+:samp:`# iocage exec examplejail1 ifconfig`
 
+.. index:: VIMAGE_VNET
+.. _VIMAGEVNET:
 
 VIMAGE/VNET
 -----------
 
-Stability: VIMAGE is considered experimental, unexpected system crashes can occur (for details please see known issues section)
+VNET is considered experimental. Unexpected system crashes
+can occur. More details about issues with VNET are available in the
+:ref:`Known Issues` section of this documentation.
 
-System requirements
-+++++++++++++++++++
+There are a number of required steps when configuring a jail to use
+VNET:
 
 **Kernel**
 
-Rebuild the kernel with the following options:
+.. tip:: If not required, disable SCTP.
 
-*(also disable SCTP if not required)*
+Rebuild the kernel with these options:
 
-::
+.. code-block:: none
 
-  nooptions       SCTP   # Stream Control Transmission Protocol
-  options         VIMAGE # VNET/Vimage support
-  options         RACCT  # Resource containers
-  options         RCTL   # same as above
+   nooptions       SCTP   # Stream Control Transmission Protocol
+   options         VIMAGE # VNET/Vimage support
+   options         RACCT  # Resource containers
+   options         RCTL   # same as above
 
 **/etc/rc.conf**
 
-Add bridge configuration to /etc/rc.conf:
-
-*(on the host node)*
+On the host node, add this bridge configuration to :file:`/etc/rc.conf`:
 
-::
+.. code-block:: none
 
-  # set up two bridge interfaces for iocage
-  cloned_interfaces="bridge0 bridge1"
+   # set up two bridge interfaces for iocage
+   cloned_interfaces="bridge0 bridge1"
 
-  # plumb interface em0 into bridge0
-  ifconfig_bridge0="addm em0 up"
-  ifconfig_em0="up"
+   # plumb interface em0 into bridge0
+   ifconfig_bridge0="addm em0 up"
```

Comment:
Some not about replacing `em0` with their network adapter should probably be here and everywhere else it's mentioned.